### DISCUSSION TITLE
Giscus cannot be displayed after adding privacy

### DISCUSSION THREAD

**Discussion by w1ndys** [squidfunk/mkdocs-material#6919] [Q&A] [answered]
Problem background：https://github.com/squidfunk/mkdocs-material/discussions/6918#/

I had a new bug. After solving the above-mentioned problem, an indescribable problem appeared in my giscus comment area. After checking the network log, I found that it was the privacy plug-in that caused some css and js of giscus to be unable to be saved to the site. ~Running normally, I have now solved this problem by `assets: false`, but I don’t know if any other functions are affected by the setting of `assets: false`~

have bug
![image](https://github.com/squidfunk/mkdocs-material/assets/109416673/355fbbf7-3bf5-406a-b7b7-ad2d69fa736a)

no bug
![image](https://github.com/squidfunk/mkdocs-material/assets/109416673/0a983fce-1a4e-453d-b248-ce0e2ef95573)

__**Oh my God, I realize this is the equivalent of doing nothing**__
Is it possible to block certain resources and only have a privacy effect on resources outside the block?

**Comment by w1ndys** [comment, marked as answer]
I think I already know the answer. I found other replies in the community. I found that there is a blocking of some resources, but it is a special tool for insiders. As a layman, I think I should only weigh the switch of privacy.

**Reply by alexvoss** [reply]
Can you please link to the replies you found so that others stumbling across this thread can find them as well? As a general rule, it is also a good idea to look for the answer in the forum before opening a new thread. I know that is not easy to do consistently but try to see posting here as an opportunity to help others who have the same problem.

**Reply by w1ndys** [reply]
sure  ，here ~
https://github.com/squidfunk/mkdocs-material/discussions/5046#/
and
https://squidfunk.github.io/mkdocs-material/plugins/privacy/#config.assets_exclude
